index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+const elem = document.getElementById('menuMobile');
+const toggleMobileMenu = () => {
+  // const logoName = document.getElementById('logo');
+  // const menuButton = document.getElementById('menuBars');
```

Comment:
- [ ] As already mentioned above, let's please remove commented codes and keep our codebase clean.

Reply:
I'll do it.  Thanks!

Reply:
Addressed 👌!